assert_test.go

```diff
@@ -0,0 +1,31 @@
+package abide
+
+import "testing"
+
+var contentTypeTestCases = []struct {
```

Comment:
Since only this test would use this particular set of cases (e.g. different set if we had `contentTypeIsXML`), let's move this directly into `TestContentTypeIsJSON`. It may also be easier to use `map[string]bool` as well.